openlibrary/macros/StarRatings.html

```diff
@@ -33,5 +33,5 @@
     <span></span>
   </div>
   $if rating:
-      <button type="submit" class="star-messaging">$_("Clear my rating")</button>
+      <button type="submit" class="star-messaging" style="margin: 0 auto;">$_("Clear my rating")</button>
```

Comment:
Could you move the inline style to the class star-messaging and add it in a existing file. Also a screenshot of the change would be great :)